cmake/std/atdm/apps/sparc/SPARCTrilinosPackagesList.cmake

```diff
@@ -0,0 +1,29 @@
+SET(SPARC_Trilinos_Packages
+  Teuchos
+  Epetra
+  EpetraExt
+  AztecOO
+  Amesos
+  Stratimikos
+  Anasazi
+  Ifpack
+  ML
+  Teko
+  NOX
+  Thyra
+  Sacado
+  Tpetra
+  Belos
+  Amesos2
+  Ifpack2
+  MueLu
+  Zoltan2
+  STKTransfer
+  STKSearch
+  STKUtil
+  Kokkos
+  KokkosCore
```

Comment:
Why does this list include both Kokkos and KokkosCore?  Shouldn't Kokkos suffice to test KokkosCore?